### PR TITLE
BS-14268 404 legacy form preview

### PR DESCRIPTION
**Pull request by rbioteau**

### CODE REVIEW

File: bundles/plugins/org.bonitasoft.studio.form.preview/src-test/java/org/bonitasoft/studio/form/preview/AbstractFormPreviewInitializationTest.java [hunk 1]
```diff
@@ -1,5 +1,7 @@
 package org.bonitasoft.studio.form.preview;
```
Comment (apupier) [inline bundles/plugins/org.bonitasoft.studio.form.preview/src-test/java/org/bonitasoft/studio/form/preview/AbstractFormPreviewInitializationTest.java]: Can you add the header which was missing previously please?

File: bundles/plugins/org.bonitasoft.studio.form.preview/src/org/bonitasoft/studio/form/preview/FormPreviewOperation.java [hunk 2]
```diff
@@ -284,8 +284,9 @@ protected void undeployDeployedProcess(final AbstractProcess process, final Proc
     }
 
     protected void undeployProcess(final ProcessAPI processApi, final ProcessDeploymentInfo info) throws ProcessDefinitionNotFoundException, DeletionException {
-        if (info.getName().equals(lastProcessDeployed) && info.getVersion().equals(formPreviewInit.VERSION)) {
+        if (info.getName().equals(lastProcessDeployed) && info.getVersion().equals(AbstractFormPreviewInitialization.VERSION)) {
             try {
+                processApi.deleteProcessInstances(info.getProcessId(), 0, 10);
```
Comment (apupier) [inline bundles/plugins/org.bonitasoft.studio.form.preview/src/org/bonitasoft/studio/form/preview/FormPreviewOperation.java]: sure that this line i snot more related to BS-14269 https://bonitasoft.atlassian.net/browse/BS-14269

**Comment by apupier** [comment]
tested locally
Waiting for merge:
- CI green
- header added

**Comment by rbioteau** [comment]
Copyright added

**Pull request merged**